tests/test_external_account.py

```diff
@@ -73,28 +73,16 @@ def __init__(
         subject_token_type,
         token_url,
         credential_source,
-        service_account_impersonation_url=None,
-        service_account_impersonation_options={},
-        client_id=None,
-        client_secret=None,
-        quota_project_id=None,
-        scopes=None,
-        default_scopes=None,
-        workforce_pool_user_project=None,
+        *args,
```

Comment:
Please add tests for token info url validation. You can copy the way they are done in this file.

Reply:
Done.  I did the same for token_url and service_account_impersonation_url